### PR TITLE
Added greek strings

### PR DESCRIPTION
**Pull request by CrisBalGreece**
Added greek translations for  platform_packages_apps_DerpQuest/res

### CODE REVIEW

**Comment by CrisBalGreece** [comment]
Added greek translations

File: res/values-el/derpquest_strings.xml [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+    <string name="DESC_LIGHT_IDLE_MAINTENANCE_MAX_BUDGET">Αυτός είναι ο μέγιστος χρόνος, που θέλουμε να είναι διαθέσιμος, όταν είμαστε σε ελαφρύ ύπνο (lightly idling)</string>
+    <string name="DESC_MIN_LIGHT_MAINTENANCE_TIME">Αυτό είναι το ελάχιστο χρονικό όριο, που θα μείνουμε σε κατάσταση επιλογών (maintenance mode) μετά απόελαφρύ ύπνο</string>
+    <string name="DESC_MIN_DEEP_MAINTENANCE_TIME">Αυτό είναι το ελάχιστο χρονικό όριο, που θα μείνουμε σε κατάσταση επιλογών (maintenance mode) μετά από βαθύ ύπνο</string>
+    <string name="DESC_KEY_NOTIFICATION_WHITELIST_DURATION">Χρονικό όριο που θέλουμε να είναι μια εφαρμογή σε λευκή λίστα, η οποία διαχειρίζεται ειδοποιήσεις</string>
+    <string name="DESC_MIN_LIGHT_MAINTENANCE_TIME">This is the minimum amount of time that we will stay in maintenance mode after a light doze</string>
+    <string name="DESC_MIN_DEEP_MAINTENANCE_TIME">This is the minimum amount of time that we will stay in maintenance mode after a full doze</string>
+    <string name="DESC_KEY_NOTIFICATION_WHITELIST_DURATION">Amount of time we would like to whitelist an app that is handling a notification</string>
```
Comment (idoybh) [inline res/values-el/derpquest_strings.xml]:
```suggestion
```
All of these are defined twice

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
Same comments of the other PR apply here. Please try to compile next time.

**Review by idoybh — APPROVED** [review]

File: res/values-el/derpquest_strings.xml (hunk 1 above)
Comment (CrisBalGreece) [reply res/values-el/derpquest_strings.xml]: Take a look to my repository : https://github.com/CrisBalGreece/platform_packages_apps_DerpQuest/blob/ten/res/values-el/derpquest_strings.xml . As you see they're not there (double lines and there is no english strings inside...)

Comment (CrisBalGreece) [reply res/values-el/derpquest_strings.xml]: Ok..my mistake again..

**Pull request merged**